File: 122562305-main CS50 course CODE/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // TODO: Prompt for the height
    int n;
    do
    {
        n = get_int("Height: ");
    } while (n > 8 || n < 1);

    //TODO: create brics

    for (int i = 0; i < n; i++)
    {
        for (int j = n - 1; j > i; j--)
            printf(" ");

        for (int j = 0; j <= i; j++)
            printf ("#");

        printf ("  ");

        for (int a = 0; a <= i; a++)
            printf ("#");

        printf("\n");
    }
}

File: 122562305-main CS50 course CODE/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

n = get_int("Height: ")
while n > 8 or n < 1:
    n = get_int("Height: ")

for i in range(n):

    for j in range(n - i - 1):
        print(" ", end = "")

    for j in range(i + 1):
        print("#", end = "")

    print("  ", end = "")

    for j in range(i + 1):
        print("#", end = "")

    print("")
